README.md

```diff
@@ -87,9 +79,54 @@ linkLauncher.launch(
 At this point, Link will open, and will return the `LinkSuccess` object if the user successfully
 completes the Link flow.
 
+### LinkSuccess
+
+When a user successfully links an account or completes the transfer, the `LinkSuccess` object is
+received. It contains a list of `LinkPayload`s that represents the linked items:
+
+```kotlin
+
+private fun onLinkSuccess(result: LinkSuccess) {
+    result.payloads.forEach { payload ->
+        when (payload) {
+            is AccessTokenPayload -> /* broker connected */
+            is DelayedAuthPayload -> /* delayed authentication */
+            is TransferFinishedSuccessPayload -> /* transfer succeed */
+            is TransferFinishedErrorPayload -> /* transfer failed */
+        }
+    }
+}
+```
+
+### LinkExit
+
+When a user exits Link without successfully linking an account or an error occurs, the `LinkExit`
+object is received. It contains an optional `errorMessage`:
+
+```kotlin
+
+private fun onLinkExit(result: LinkExit) {
+    if (result.errorMessage != null) {
+        /* use error message */
+    }
+}
```

Comment:
is it a substitute to `onExit` method, if so should we harmonize it accross other SDKs as well?